Dockerfile.template

```diff
@@ -29,7 +29,9 @@ ENV GPG_KEYS \
 
 RUN set -ex; \
 	for key in $GPG_KEYS; do \
-		gpg --batch --keyserver keyserver.ubuntu.com --recv-keys "$key"; \
+                curl -sSL \
+			"http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x$key" \
+			| gpg --import; \
```

Comment:
This change is definitely unacceptable -- `gpg --recv-keys` does more actual validation of the fetched bytes than `curl` does. :grimacing: